site/installing.md

```diff
@@ -154,81 +141,20 @@ After it’s been launched, open your browser to `http://localhost:4040`.
 
 Version 2 of this YAML file supports networks and volumes as defined by any plugins you might be using. See [Compose File Reference](https://docs.docker.com/compose/compose-file/) for more information.
 
-### <a name="docker-compose-cloud"></a>Using Docker Compose with Weave Cloud
-
-**1.** First, obtain a Cloud token from Weave Cloud by signing up at [https://cloud.weave.works](https://cloud.weave.works/).
-
-!['Weave Cloud Token'](images/weave-cloud-token.png)
-
-**2.** Decide on which format version to use:
-
-**Docker Compose Format Version 1:**
-
-    probe:
-      image: weaveworks/scope:1.7.2
-      net: "host"
-      pid: "host"
-      privileged: true
-      labels:
-        - "works.weave.role=system"
-      volumes:
-        - "/var/run/docker.sock:/var/run/docker.sock:rw"
-      command:
-        - "--probe.docker=true"
-        - "--service-token=${SCOPE_SERVICE_TOKEN}"
-
-**Docker Compose Format Version 2:**
-
-    version: '2'
-    services:
-      probe:
-        image: weaveworks/scope:1.7.2
-        network_mode: "host"
-        pid: "host"
-        privileged: true
-        labels:
-          - "works.weave.role=system"
-        volumes:
-          - "/var/run/docker.sock:/var/run/docker.sock:rw"
-        command:
-          - "--probe.docker=true"
-          - "--service-token=${SCOPE_SERVICE_TOKEN}"
-
-Version 2 of this YAML file supports a network mode that may be required with any Docker plugins that you are using. See [Compose File Reference](https://docs.docker.com/compose/compose-file/) for more information.
-
-Copy the contents of one of the above fragments into a file `docker-compose.yml`.
-
-**3.** Once you've set up a compose file, launch Scope onto your local machine:
-
-    SCOPE_SERVICE_TOKEN=<token>  docker-compose up -d
-
-Where,
-
-* `SCOPE_SERVICE_TOKEN=<token>` is the token you obtained after you signed up for Weave Cloud.
-
-Note that you will need to launch Scope on every node that you want to monitor.
+## <a name="orchestrators"></a>Installing on Orchestrators
 
-**4.** Go to [https://cloud.weave.works](https://cloud.weave.works) and click 'View Instance'.
+### <a name="k8s"></a>Kubernetes
 
-## <a name="k8s"></a>Installing Scope on Kubernetes
+If your cluster is on GKE, you first need to grant the requisite
+permissions for installation:
 
```

Comment:
If your cluster is on GKE, you first need to grant the requisite
permissions for installation:

change to: 

If your cluster is on GKE, first you need to grant permissions for the installation with:

Reply:
Fixing.